test/api/junction_detection_test.cpp

```diff
@@ -14,12 +14,20 @@
 //     EXPECT_EQ(expected, std_cout);
 // }
 
+// Reference\tm2257/8161/CCS\t41972616\tForward\tRead \t0\t2294\tForward\tchr21
+// INS from Primary Read - Sequence Type: Reference; Sequence Name: m2257/8161/CCS; Position: 41972616; Orientation: Reverse
+//                         Sequence Type: Read; Sequence Name: 0; Position: 3975; Orientation: Reverse
+//                         Chromosome: chr21
+// Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\tm41327/11677/CCS
+// BND from SA Tag - Sequence Type: Reference; Chromosome: chr22; Position: 17458417; Orientation: Forward
+//                   Sequence Type: Reference; Chromosome: chr21; Position: 41972615; Orientation: Forward
+//                   Sequence Name: m41327/11677/CCS
+
```

Comment:
Why is this here?

Reply:
I thought I'd add an explanation of the string as I stumbled over it and was confused as to what the parts of the string were.

Reply:
Ok, makes sense :)

Reply:
I'm confused by this string:

> // Reference\tm2257/8161/CCS\t41972616\tForward\tRead \t0\t2294\tForward\tchr21

It should consist of breakend1, breakend2 and the read name it was detected from. However, the read name here is `chr21` and the reference chromosome of breakend1 is `m2257/8161/CCS\t41972616`. Those are swapped but I don't immediately see why 😕